lib/builder/util.go

```diff
@@ -63,15 +65,25 @@ func checkBuildEnv() error {
 	if err != nil {
 		return trace.Wrap(err)
 	}
-	_, err = client.Version()
-	if err != nil {
-		log.Error(trace.DebugReport(err))
-		return trace.BadParameter(
-			`Docker does not seem to be available on this machine.
+	dockerErr := trace.BadParameter(
+		`Docker does not seem to be available on this machine.
 To resolve the issue:
   * Install Docker (https://docs.docker.com/engine/installation/).
   * Make sure it can be used by a non-root user (https://docs.docker.com/install/linux/linux-postinstall/).`)
+
+	_, err = client.Version()
+	if err != nil {
+		log.Error(trace.DebugReport(err))
+		return dockerErr
+	}
+	_, err = client.Version()
+
+	err = utils.Exec(exec.Command("docker", "help"), ioutil.Discard)
+	if err != nil {
+		log.Error(err)
```

Comment:
`log.WithError(err).Error("Failed to ...")`

Reply:
Fixed in 2ae2d38.